.github/workflows/system-tests.yml

```diff
@@ -357,9 +375,15 @@ jobs:
           - runner
           - agent
           - weblog-rack
-#           - weblog-sinatra14 # DEV-2.0: Uncomment after https://github.com/DataDog/system-tests/pull/1882 is merged
+          - weblog-sinatra14
           - weblog-sinatra20
           - weblog-sinatra21
+          - weblog-sinatra22
+          - weblog-sinatra30
+          - weblog-sinatra32
+          - weblog-sinatra32
```

Comment:
this line is duplicated

Reply:
Thanks ! The first one is supposed to be weblog-sinatra31